operator/pkg/controller/cassandradatacenter/cassandradatacenter_controller.go

```diff
@@ -146,12 +146,36 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 			return requests
 		})
 
+	nodeTaintsChangedPredicate := predicate.Funcs{
```

Comment:
very much a nit - this is going to be our predicate for all Node UpdateEvents, right? so I might change this name

Reply:
I named the predicate what it is checking for. Much like the GenerationChangedPredicate and managedByCassandraOperatorPredicate predicates.